### ISSUE TITLE
SortedSwitchCases Bug?

### ISSUE THREAD

**Issue by PompeiaPaetenari** [nicklockwood/SwiftFormat#815]
I am sure you are familiar with the age 'ol alphabetic sorting on numbers not working out so well... I suspect this is something we can account for here?
There is only one of these in our code, but it stood out when I made the PR with this rule being enforced and I figured I better ask about it, too.  I would guess we can see if the variable we are switching on is numeric and sort numerically, else we sort as it does now?

```swift
func numberOfDaysPerMonth(month: Int) -> Int {
  switch month {
    case 1, 3, 5, 7, 8, 10, 12:
      return 31
```
became
```swift
func numberOfDaysPerMonth(month: Int) -> Int {
  switch month {
    case 1, 10, 3, 5, 7, 8, 12:
      return 31
```

**Comment by nicklockwood** [comment]
👍 I'll get that fixed

**Comment by PompeiaPaetenari** [comment]
By the way, we are a bit baffled how `12` remained at the end and did not get moved, too, assuming a standard ASCII sorting.  I glanced at the code for this and it appears to grab the last token next to the colon, but someone else asked if it is anchored.  I can only speculate it has to do with the range tokens around `nextStart` and `nextEnd` because I mess up that stuff all the time.

**Comment by nicklockwood** [comment]
@PompeiaPaetenari fixed in 0.47.5.
